templates/index.html

```diff
@@ -23,6 +23,9 @@
 #}
 
 {% block takeover_content %}
+{% comment %}
+  {% include "takeovers/_fin-serv-whitepaper-takeover.html" %}
+{% endcomment %}
```

Comment:
We don't need this. We can add then releasing.

Reply:
the issue is that a designer can't test it easily if we don't make it available, so I ended up including it.